dashboard/config/scripts/test_longassessment_multigroup_split.level_group

```diff
@@ -0,0 +1,16 @@
+name 'Test longassessment levelgroup split'
+submittable 'true'
+
+page 'First page'
+level 'K-1 Happy Maps Multi 1'
+level 'Multi Horizontal Scroll Test'
+level 'Choose 2 Test'
+
+page 'Second page'
+level 'K-1 Plant a Seed Multi 1'
+level 'K-1 Plant a Seed Multi 2'
+level  'K-1 Artist 1 Multi 1'
+
+page 'Third page'
```

Comment:
Not yet, but the comps show one form of title, and I think it'd be a neat feature.  But sure, I can leave these blank for now.